### ISSUE TITLE
Requirements to get a job?

### ISSUE THREAD

**Issue by Kl0tzfaust** [edenspiekermann/ama#2]
What are the requirements to get a job as a graphic designer at edenspiekermann?

Many companies want an employee with a minimum of 20 years work experience, 3 PhDs, Master of Design, professional knowledge of all programming languages, prepared to work 24/7, able to speak English, German, Spanish, French, Chinese, Arabic - fluently, excellent communication skills - written and verbal, and so on..

What are the minimum requirements to work at edenspiekermann? Do people who got just out of Graphic Design school and new to the job market even have a slightly chance to get a job as a graphic designer?

I am looking forward to the reply

**Comment by harryfk** [comment]
Apparently my last comment came over as a little trollish, so I decided to make up for it and provide an honest answer this time. Please take into account that I left the Berlin office almost a year ago, but I worked there for four years in the digital team, assuming pretty much every role you can have on a digital project at some point. I loved coming to work almost every day and thoroughly enjoyed being part of this team. Everybody was always super welcoming, helpful and genuinely cared about the projects and the company. People of all skill levels started and contributed … there were no intern-only tasks, instead everyone was a full member of the team, doing their best work and getting a lot of freedom to learn new skills. Job titles or disciplines didn’t matter at all and there was never any blaming if something went wrong. I almost never worked long hours either.

There were no minimum requirements to work there and interviews focused very much on the person, not on skills. As Erik always said: Skills can be learnt, but an asshole stays an asshole. Keep in mind: Job postings usually describe an ideal, so don’t worry if you don’t tick all the boxes. It even says that on their jobs page, usually towards the end, e.g. [here](http://www.edenspiekermann.com/jobs/berlin-frontend-developer-support-qa-m-f) (“We know that nobody’s perfect …” – I wrote this back in the day and I am sure it still holds up today).

Bottom line: Stop asking questions and apply already. :)

**Comment by Kl0tzfaust** [comment]
Thank you harryfk for your honest and detailed thoughts!

Nice to see that edenspiekermann cares more about the people and not only the skills. 
It seems like the whole agency is like a family where everyone is treated the same (at least how you describe it), i would love to work there! :)

So thanks again, it is really helpful! 

PS: i hope they are hiring graphic designers in 2 years, then i will apply for sure ;)

**Comment by martzoukos** [comment]
Whatever Harry has said is true.

Regarding your situation, specifically, coming out of college, a very good start is to take an internship job (which is being payed by most companies in Berlin, if I’m not mistaken) and is the best way to get you to **know** the actual job by doing it. After finishing it then you immediately have a track record in your resume that says that you can be committed to a project, know how to work in a real, and not just academical, team setting and can...just ship stuff.
And, if you are a good person, then no matter how many years you have in the field, this is what companies will keep asking for, not Arabic :).

So, when you decide to apply in 2 years, I’m sure that Edenspiekermann will have many designer job openings but along with them there will plenty of internship positions.

Tip: you can actually [apply for an internship position](http://www.edenspiekermann.com/jobs) while you study (plenty of our colleagues are/were still studying). This will give you a **big** headstart.

**Comment by HugoGiraudel** [comment]
On a very general level, I would say not to worry too much about requirements to join a company. Especially in Berlin, as far as I can tell. Companies seem more interested in hiring profiles based on experience and mostly *character* than actual diplomas.

I come from France. It matters there. Almost everywhere. Diplomas are everything. You don’t have it, you are not worth hiring. At least not for interesting positions. It gets better with the years, but when you are a fresh (non-)graduate, it is hard to find a decent job without have a computer science degree or something similar.

If you do not have diplomas (yet or at all), try to show that you care about your job and that you like it. I believe this is what companies are looking for: involved and caring people. Contribute to open-source projects. Make yourself a good portfolio. Show that you are open-minded and prone to onboard on new projects. And more importantly, have a nice, honest and friendly interview. That's the first step towards getting hired. :)

**Comment by Kl0tzfaust** [comment]
Thanks Hugo for your thoughts, its very inspirational!
You should become a motivation coach ;)

**Comment by HugoGiraudel** [comment]
You know what? I should. I'll ask the bosses at ESPI if I can change my job title. :)

**Comment by sdietrichberlin** [comment]
"International Vice President Digital Talent Engagement"?
